tests/test_main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+from unittest import mock
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_do_nothing(prediction):
+    prediction.return_value = 10
+    assert cryptocurrency_action(10) == "Do nothing"
+    prediction.return_value = 9.6
+    assert cryptocurrency_action(10) == "Do nothing"
+    prediction.return_value = 9.5
+    assert cryptocurrency_action(10) == "Do nothing"
+    prediction.return_value = 10.4
+    assert cryptocurrency_action(10) == "Do nothing"
+    prediction.return_value = 10.5
+    assert cryptocurrency_action(10) == "Do nothing"
```

Comment:
Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.